kevin_li/week05/day1/games_on_rails/app/controllers/games_controller.rb

```diff
@@ -0,0 +1,44 @@
+class GamesController < ApplicationController
+
+  def index
+  end
+
+  def magic_8_ball
+    @answers = [
+      "It is certain.",
+      "It is decidedly so.",
+      "Without a doubt.",
+      "Yes definitely.",
+      "You may rely on it.",
+      "As I see it, yes.",
+      "Most likely.",
+      "Outlook good.",
+      "Yes.",
+      "Signs point to yes.",
+      "Reply hazy try again.",
+      "Ask again later.",
+      "Better not tell you now.",
+      "Cannot predict now.",
+      "Concentrate and ask again.",
+      "Do not count on it.",
+      "My reply is no.",
+      "My sources say no.",
+      "Outlook not so good.",
+      "Very doubtful."
+    ]
+
+    @answer = @answers[Random.rand(20)]
+
```

Comment:
Consider `@answer = @answers.sample`

Also, since `@answers` is not being used in your template, you should just keep it a local variable, i.e. `answers`.